src/main/java/exercises/LinkedList.java

```diff
@@ -11,7 +11,11 @@
 }
 
 public class LinkedList {
-    Node first = null;
+    Node first;
+
+    public LinkedList() {
```

Comment:
Why did you add a default constructor?

Reply:
This is actually a special case, so there is a good reason for doing it this way. ;-)